connect/workflows/core.py

```diff
@@ -280,47 +286,43 @@ async def run(self) -> Optional[dict]:
             logger.trace(f"Running {self.__class__.__name__}")
             await self.transform()
             await self.persist()
-            results = await self.transmit()
+            result = await self.transmit()
             await self.synchronize()
-            return {"lfh_message": self.message, "transmit_result": results}
+            return self._set_response(result)
```

Comment:
From the workflow run() method, return the result (either the LFH message or transmit response) that can be returned directly by the FastAPI route.  If we next want to parameterize the response behavior, we can do that transparently to the caller in _set_response().